src/utils.js

```diff
@@ -77,15 +77,15 @@ var utils = {
 		};
 	},
 
-	// @todo move this method in Chart.helpers.options.toFont
+	// @todo move this method in Chart.helpers.toFont
```

Comment:
I think `Chart.helpers.toFont` can be used instead of this.

[helpers.options.js#L110-L141](https://github.com/chartjs/Chart.js/blob/e00dc1e9147057db0e1b034f060440fa983b478e/src/helpers/helpers.options.js#L110-L141)